src/Form/RedirectForm.php

```diff
@@ -76,7 +76,7 @@ public function form(array $form, FormStateInterface $form_state) {
         '#type' => 'language_select',
         '#title' => t('Language'),
         '#languages' => Language::STATE_ALL,
```

Comment:
Simply adding `translatable = TRUE` to the entity annotation sort of blows things up (claims the `hash` column doesn't exist). This has to do with the index that is created since I think marking fields/entities as translatable moves things into separate tables.